modules/ROOT/pages/push/index.adoc

```diff
@@ -46,21 +30,13 @@ WARNING: It is highly recommended to not sent any send sensitive personal or con
 For analytic purposes on our link:#_dashboard[Dashboard] we store the content of the +alert+ key sent to the UnifiedPush Server. The content of the +alert+ key belongs to the metadata, which is deleted after 30 days, using a nightly job within the UnifiedPush Server.
 
 
-=== How the UnifiedPush Server Works
+== How the UnifiedPush Server Works
 
-The AeroGear UnifiedPush Server offers a unified Notification Service API to the above mentioned Push Network Services. When a push message request is sent to the UnifiedPush Server, it is internally translated into the format of these 3rd party networks. This gives a server the ability to send Push notifications to different mobile platforms.
+The AeroGear UnifiedPush Server offers a unified Notification Service API to the above mentioned Push Network Services.
```

Comment:
The AeroGear "UnifiedPush Server" should be "{unifiedpush-service}" here?

Reply:
yep